### DISCUSSION TITLE
cool possible fetures

### DISCUSSION THREAD

**Discussion by munozajj** [tryallthethings/DNS-Swapper#11] [Ideas]
It would be cool to extend its functionality so that you could swap:

IPs
subnet mask
default gateway

and in the settings you could select with clicking the icon does (swap ip/swap gateway/swap dns)

**Comment by tryallthethings** [comment]
Thank you for the idea, but I feel like this is way out of scope for this tool. I would have to rebuild it completely to support this. And there are already other tools such as NetSetMan out there that already do this. DNS-Swapper was meant to be a small tool to switch between an Ad-Blocking DNS (such as Pi-Hole) and an open one.
